src/artemis_driver.cpp

```diff
@@ -111,6 +112,12 @@ TaskListStatus ArtemisDriver<GEOM>::Step() {
   if (do_radiation) status = IMC::JaybenneIMC<GEOM>(pmesh, tm.time, tm.dt);
   if (status != TaskListStatus::complete) return status;
 
+  // Execute operator split physics
+  for (auto &fn : OperatorSplitTasks) {
+    status = fn(pmesh, tm, integrator->dt).Execute();
+    if (status != TaskListStatus::complete) return status;
+  }
+
```

Comment:
I would prefer that instead of all this OperatorSplitTasks boilerplate throughout the code, we just add the CoagulationTasks directly into the artemis driver. That will be much cleaner and clearer.

Reply:
Looks like @adamdempsey90 had same thought as my above comment.

Reply:
I changed as you suggested in the new updated code